.github/workflows/ci.yaml

```diff
@@ -10,22 +10,11 @@ jobs:
     container:
       image: docker://bitcli/bit:latest
     steps:
-      - name: Bit config
-        run: |
-          bit config set interactive false
-          bit config set analytics_reporting false
-          bit config set anonymous_reporting false
-          bit config set user.token ${{ env.BIT_TOKEN }}
-          bit config set no_warnings true
-          bit config set error_reporting true
-          bit config set user.name symphony_backend
+      - uses: teambit/setup-action@v1
```

Comment:
are you sure this can come before actions/checkout?

Reply:
it seems to work fine on the 2 other repositories that use it 🙂 what do you think could be the issue there?